### PR TITLE
troubleshooting failed domain transfers. updating instructions

### PR DESCRIPTION
**Pull request by jason-cf**

### CODE REVIEW

File: content/registrar/troubleshooting.md [hunk 1]
```diff
@@ -36,6 +36,11 @@ Some registrars may prohibit transfer requests if you have WHOIS privacy service
 
 ## Restart your transfer
 
-1.  Log in to [Cloudflare dashboard](https://dash.cloudflare.com/login), and select your account and domain.
-2.  In the **Overview** app for your domain, scroll down to **Domain Registration.**
-3.  Select **Cancel and Retry**. Once you initiate the retry, reenter your auth code and confirm your WHOIS information.
+{{<Aside type="note">}}
+This does not apply to .uk domains.
```
Comment (marciocloudflare) [inline content/registrar/troubleshooting.md]:
```suggestion
This solution does not apply to `.uk` domains.
```

File: content/registrar/troubleshooting.md [hunk 2]
```diff
@@ -36,6 +36,11 @@ Some registrars may prohibit transfer requests if you have WHOIS privacy service
...
+{{</Aside>}}
+
+1. Log in to [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account.
+2. Select **Domain Registration** > **Manage Domains**.
+3. On the right of the domain name, select **Manage**.
```
Comment (marciocloudflare) [inline content/registrar/troubleshooting.md]:
```suggestion
3. Find the correct domain and select **Manage**.
```

Comment (marciocloudflare) [reply content/registrar/troubleshooting.md]: We should avoid writing out locations like this, in case the dash changes things around

**Review by marciocloudflare — APPROVED** [review]

File: content/registrar/troubleshooting.md (hunk 2 above)
Comment (jason-cf) [reply content/registrar/troubleshooting.md]: Agree!

**Pull request merged**